test/Webfactory/Slimdump/Database/DumperTest.php

```diff
@@ -83,9 +87,36 @@ public function testDumpDataWithFullConfiguration()
 
         $table = new Table(new \SimpleXMLElement('<table name="test" dump="full" />'));
 
-        $dumper->dumpData('test', $table, $this->dbMock);
-        $output = $outputBuffer->fetch();
+        $this->dumper->dumpData('test', $table, $this->dbMock);
+        $output = $this->outputBuffer->fetch();
 
         $this->assertContains('INSERT INTO', $output);
     }
+
+    public function testDumpTriggers()
```

Comment:
I have improved tests and also included one to make sure it correctly iterates all triggers. I don't think we need a variation of this for the "no triggers defined" case.